cogs/channel_mods.py

```diff
@@ -853,16 +853,16 @@ async def modlog(self, ctx, id_in):
 
     @modlog.command(name='delete', aliases=['del'])
     @hf.is_admin()
-    async def modlog_delete(self, ctx, user, *, index):
+    async def modlog_delete(self, ctx, user, *, indices=""):
         """Delete modlog entries.  Do `;modlog delete user -all` to clear all warnings from a user.
         Alias: `;ml del`"""
+        if not indices:  # If no index is given:
+            await hf.safe_send(ctx, "Please write numeric indices equal to or greater than 1 or "
+                                    "`-all` to clear the user's modlog.")
+            return
```

Comment:
Pulled this up from the end of the command to the front just to get it out of the way. Now we count up: "what if there's 0 indices, else if there's one index, else if there's more than 1", then end